tests/e2e/Services/GraphQL/Base.php

```diff
@@ -1929,6 +1931,33 @@ functionsCreateBuild(functionId: $functionId, deploymentId: $deploymentId, build
                         }
                     }
                 }' . PHP_EOL . self::$FRAGMENT_ATTRIBUTES;
+            case self::$CREATE_PROVIDER:
+                return 'mutation createProviderMsg91(
+                    $providerId: String!,
+                    $name: String!,
+                    $from: String!,
+                    $senderId: String!,
+                    $authKey: String!
+                    $default: Boolean,
+                    $enabled: Boolean,
+                  ) {
+                    messagingCreateProviderMsg91(
+                      providerId: $providerId,
+                      name: $name,
+                      from: $from,
+                      senderId: $senderId,
+                      authKey: $authKey
+                      default: $default,
+                      enabled: $enabled,
+                    ) {
+                      _id
+                      name
+                      provider
+                      type
+                      default
+                      enabled
+                    }
+                  }';
```

Comment:
Let's make sure all the new routes are covered in GraphQL tests as well

Reply:
added graphql tests